index.js

```diff
@@ -83,7 +125,15 @@ function countByRating() {}
       // Toy Story 4
     };
  */
-function findById() {}
+function findById(movies, id) {
+  let nope = null
+  for (let i = 0; i < movies.length; i++){
```

Comment:
You know what I'm going to say now right? Yup `for...of`. I'm going to stop mentioning it because I think you probably picked it up by now.